plugins/CoreUpdater/SystemSettings.php

```diff
@@ -52,19 +53,20 @@ protected function init()
     {
         $this->title = Piwik::translate('CoreAdminHome_UpdateSettings');
 
-        $isWritable = Piwik::hasUserSuperUserAccess() && CoreAdminController::isGeneralSettingsAdminEnabled();
+        $isWritable = Piwik::hasUserSuperUserAccess()
+                        && CoreAdminController::isGeneralSettingsAdminEnabled()
+                        && SettingsPiwik::isMultiServerEnvironment() === false;
         $this->releaseChannel = $this->createReleaseChannel();
         $this->releaseChannel->setIsWritableByCurrentUser($isWritable);
 
-        $isWritable = $isWritable && PluginUpdateCommunication::canBeEnabled();
-        $this->sendPluginUpdateEmail = $this->createSendPluginUpdateEmail();
-        $this->sendPluginUpdateEmail->setIsWritableByCurrentUser($isWritable);
-
-        $isWritable = Piwik::hasUserSuperUserAccess() && CoreAdminController::isGeneralSettingsAdminEnabled();
         $dbSettings = new Settings();
         if ($isWritable && $dbSettings->getUsedCharset() !== 'utf8mb4' && DbHelper::getDefaultCharset() === 'utf8mb4') {
             $this->updateToUtf8mb4 = $this->createUpdateToUtf8mb4();
         }
+
+        $isWritable = $isWritable && PluginUpdateCommunication::canBeEnabled();
+        $this->sendPluginUpdateEmail = $this->createSendPluginUpdateEmail();
+        $this->sendPluginUpdateEmail->setIsWritableByCurrentUser($isWritable);
```

Comment:
Actually only the release channel is a configuration that is managed in config. The other values are "normal" settings, stored in database. So for those it wouldn't hurt to have them writable.

Reply:
Thank you for that, I have updated the code to reflect that (and did a very minor refactor to remove duplicate code).